pom.xml

```diff
@@ -343,7 +343,7 @@
       <versionx.org.jgroups.aws.jgroups-aws>2.0.1.Final</versionx.org.jgroups.aws.jgroups-aws>
       <versionx.org.jgroups.azure.jgroups-azure>2.0.2.Final</versionx.org.jgroups.azure.jgroups-azure>
       <versionx.org.jgroups.jgroups>${version.jgroups}</versionx.org.jgroups.jgroups>
-      <versionx.org.jgroups.jgroups-raft>${version.jgroups.raft}</versionx.org.jgroups.jgroups-raft>
+      <versionx.org.jgroups.jgroups-raft>1.0.12.Final</versionx.org.jgroups.jgroups-raft>
```

Comment:
Hm, shouldn't this change apply to `build/configuraiton/pom.xml` instead ?

Reply:
Argh, yeah, trusting too much on dependabot. Updating.